package.json

```diff
@@ -51,7 +51,7 @@
     "omit.js": "^1.0.0",
     "prop-types": "^15.5.7",
     "rc-animate": "^2.4.1",
-    "rc-calendar": "~9.4.0",
+    "rc-calendar": "~9.5.0",
```

Comment:
it's a irrelevant change ? and the `rc-pagination`

~9.4.0: >=9.4.0 && < 9.5.0

Reply:
I have added Kurdish translation to rc-calender and rc-pagination. Please check this https://github.com/ant-design/ant-design/pull/9027#issuecomment-359361333 and this https://github.com/ant-design/ant-design/pull/9015#discussion_r162527232